HerPublicWebsite.BusinessLogic/Models/LocalAuthorityData.cs

```diff
@@ -300,7 +300,7 @@ private enum IncomeThreshold
         { "530", new LocalAuthorityDetails("South Cambridgeshire District Council", Hug2Status.Live, "http://www.scambs.gov.uk/", IncomeBandOptions[IncomeThreshold._34500]) }, // Part of Cambridge City Council Consortium
         { "1040", new LocalAuthorityDetails("South Derbyshire District Council", Hug2Status.Pending, "https://www.southderbyshire.gov.uk", IncomeBandOptions[IncomeThreshold._31000]) },
         { "119", new LocalAuthorityDetails("South Gloucestershire Council", Hug2Status.Live, "http://www.southglos.gov.uk/", IncomeBandOptions[IncomeThreshold._31000]) },
-        { "1125", new LocalAuthorityDetails("South Hams District Council", Hug2Status.Pending, "http://www.southhams.gov.uk/", IncomeBandOptions[IncomeThreshold._31000]) },
+        { "1125", new LocalAuthorityDetails("South Hams District Council", Hug2Status.Live, "http://www.southhams.gov.uk/", IncomeBandOptions[IncomeThreshold._31000]) },
```

Comment:
South Hams District Council is also not mentioned on [PC-615](https://beisdigital.atlassian.net/browse/PC-615).

Reply:
Same explanation as above 😄